src/components/grid/grid-item/grid-item.style.js

```diff
@@ -46,8 +48,9 @@ const GridItemStyle = styled.div`
   margin: 0;
 
   ${flexbox}
```

Comment:
So you should remove `${flexbox}` and do what you've got for the grid props, i.e.: 

```jsx
${({ justifySelf }) => flexbox({ justifySelf })}
${({ alignSelf }) => flexbox({ alignSelf })}
```